corona-warn-app/cwa-app-android
# Root Detection & User Information (EXPOSUREAPP-9069)

This PR addresses EXPOSUREAPP-9069.

Description: "As CWA user I want to get informed if my device is rooted, so that I'm aware that other apps may be able to access CWA data".

How to test:
- Root your device / or get a rooted device
- Start the app
- Check if a notification with relevant root information appear

- [x] add missing links

Screenshot:
![image](https://user-images.githubusercontent.com/46747780/135272952-0ac6572d-b6c3-46e0-b643-496d18f46398.png)
